projects/resource-tags.md

```diff
@@ -0,0 +1,103 @@
+
+* [Define the scoped set of resource attributes and their recommended values]
+* [Ensure alignment with existing OpenTelemetry resource attributes where applicable]
+* [Collaborate with the open source community and other cloud providers for cross-domain alignment]
+
+---
+## Initial Attribute Scope and Proposals
+This section outlines the initial set of resource-level attributes the working group will define, along with their expected types, example values, and rationale. Where applicable, we build on existing OTel attributes (e.g., deployment.environment); in other cases, we propose new attributes or namespaces to fill key gaps in representing organizational and data context.
+
+## Initial Attribute Scope
+| Attribute              | Allowed Values                                                             |
+|------------------------|----------------------------------------------------------|
+| deployment.environment | "production", "staging", "development", "testing"                         |
+| service.name           | Dynamic values (e.g., "my-application")                                   |
+| owner                  | Dynamic values (e.g., "team@example.com", "team-id")                      |
+| cost_center            | Dynamic values (e.g., "cc-1234", "us-west-finance")                       |
+| business_unit          | Dynamic values (e.g., "marketing", "sales")                               |
+| data_sensitivity       | "high", "medium", "low"                                                   |
+| criticality            | "mission_critical", "high", "medium", "low"                               |
+                   
+---
+
+## Deliverables
+Initial deliverables will include:
+
+* A specification defining semantic conventions for the scoped set of resource attributes: environment, application, owner, cost_center, business_unit, criticality and data_sensitivity
+* Recommended value sets and usage guidance for applicable resource attributes 
+* Alignment with existing OpenTelemetry resource attributes where relevant
+* Documentation and examples for adoption by cloud providers, observability platforms, and security tools
+
+---
+## Staffing / Help Wanted
+
+We are seeking domain experts to work on the definition, alignment, and adoption of standardized resource attributes across cloud platforms and observability systems.
+
+The goal is to follow @tedsuo's proposed [Semantic Convention Process](https://docs.google.com/document/d/1ghvajKaipiNZso3fDtyNxU7x1zx0_Eyd02OGpMGEpLE/edit#heading=h.xc2ft2cddhny), with the following stages:
+
+- **Stage 1: Working Group Preparation** — Define scope, gather contributors, and align on the initial set of attributes.
+- **Stage 2: Prototyping and Finalizing Semantic Conventions Proposal** — Build prototypes using the proposed attributes and refine the semantic conventions for final review and submission.
+- **Stage 3: Implementation** 
+
+---
+### Required staffing
```

Comment:
This will need to list role as well.

- You should list project leads (yourself, of course)
- There should be a "Semantic Convention Maintainers" sections, trask and I can be in that section.
- There should be a "GC sponsor" - @trask would that be you?
- There should be a "TC sponsor" with a sponsorship level.  If this is a "escalating" sponsorship I can be that sponsor, but I am at my limit if this requires a higher level sponsorship (we can discuss requirement in TC).
- Other sponsors should be listed in terms of what expertise they bring or where they contribute within OpenTelemetry.

Reply:
Regarding sponsorship, please see:

- https://github.com/open-telemetry/community/blob/main/project-management.md#project-proposal
- https://github.com/open-telemetry/community/blob/main/tech-committee-charter.md#sponsorship-requirements
- https://github.com/open-telemetry/community/blob/main/governance-charter.md#regular-check-ins-with-sig-leads